lib/AstToMiniRust.ml

```diff
@@ -488,7 +492,17 @@ let rec translate_type_with_config (env: env) (config: config) (t: Ast.typ): Min
   | TArrow _ ->
       let t, ts = Helpers.flatten_arrow t in
       let ts = match ts with [ TUnit ] -> [] | _ -> ts in
-      Function (0, List.map (translate_type_with_config env config) ts, translate_type_with_config env config t)
+      let translate_parameter_type t =
+        match translate_type_with_config env config t with
+        | Ref (_, m, (Slice (Name (t, _ :: _)) as slice_t)) when is_contained t ->
+            (* Unlike top-level declarations (where we use 'b), we omit the lifetime name here.
+               Writing an explicit lifetime name here would also require adding a for<'b>.
+               By omitting the lifetime name, Rust will automatically do this. *)
```

Comment:
FWIW, we could also omit the explicit `'b` lifetime for top-level declarations as well.